modules/core/src/main/scala/com/azavea/stac4s/core/ItemCollection.scala

```diff
@@ -1,4 +1,4 @@
-package com.azavea.stac4s
```

Comment:
I'm wondering if these should actually be `core` or moved out. I [switched](https://github.com/azavea/stac4s/pull/5) the package name previously but didn't move the files. I found putting everything in `core` was kind of annoying when using the library.

I can be convinced the other way since that was just my experience. In my opinion adding `core` doesn't add much clarity since the library is a bunch of types essentially.

Reply:
@notthatbreezy SGTM; will move it back from the core